test/unit/controllers/declaration/CommodityDetailsControllerSpec.scala

```diff
@@ -138,15 +139,48 @@ class CommodityDetailsControllerSpec extends ControllerSpec with OptionValues {
     }
 
     onJourney(CLEARANCE) { request =>
-      "return 303 (SEE_OTHER) and redirect to UN Dangerous Goods Code page" in {
-
-        withNewCaching(request.cacheModel)
+      def controllerRedirectsToNextPageForProcedureCodeAndExsStatus(procedureCode: String, exsStatus: String, call: Call) = {
```

Comment:
Nit: Could you change the name to `expectedCall` (I assume this is this field's purpose).